app/email/email.go

```diff
@@ -741,33 +741,6 @@ func (es *Service) CreateVerifyEmailToken(userID string, newEmail string) (*mode
 	return token, nil
 }
 
-func (es *Service) SendAtUserLimitWarningEmail(email string, locale string, siteURL string) (bool, error) {
```

Comment:
Could you update mocks with `make email-mocks`?

Reply:
I forgot to update. There's an issue with that make target
```log
go install github.com/vektra/mockery/...@v1.1.2
/Users/allanguwatudde/Desktop/MATTERMOST/mattermost-server/bin/mockery -dir app/email --name ServiceInterface -output app/email/mocks -note 'Regenerate this file using `make email-mocks`.'
2022/03/30 12:39:29 internal error: package "bytes" without types was imported from "github.com/mattermost/mattermost-server/v6/app/email"
make: *** [email-mocks] Error 1
```

Reply:
hmm, not sure why. It currently works on my machine using `go version go1.16.5 linux/amd64` . Quick websearching suggests might be something with the go version used? https://github.com/ent/ent/issues/2155 , https://github.com/golang/go/issues/37617

```
make email-mocks
go install github.com/vektra/mockery/...@v1.1.2
/home/n/rw/mattermost-server/bin/mockery -dir app/email --name ServiceInterface -output app/email/mocks -note 'Regenerate this file using `make email-mocks`.'
Generating mock for: ServiceInterface in file: app/email/mocks/ServiceInterface.go
```